demos/projects/PC/linux/ADU.md

```diff
@@ -0,0 +1,210 @@
+# Use Linux to simulate an over the air update using Azure IoT Middleware for FreeRTOS
+
+This sample will allow you to run the Azure Device Update (ADU) flow, used to update devices over the air (OTA). The executable will not write the new update to the computer, only download and simulate applying it. The ADU flow is otherwise run as an updatable device would run. The following is an outline of the steps to run this sample.
+
+- [Prerequisite Note](#prerequisite-note)
+- [Get the middleware](#get-the-middleware)
+- [Install Prerequisites](#install-prerequisites)
+- [Prepare the simulation](#prepare-the-simulation)
+  - [Set the Virtual Ethernet Interface](#set-the-virtual-ethernet-interface)
+- [Build the image](#build-the-image)
+- [Confirm simulated device connection details](#confirm-simulated-device-connection-details)
+- [Prepare the ADU Service](#prepare-the-adu-service)
+- [Deploy the Over the Air Update](#deploy-the-over-the-air-update)
+  - [Build the Update Image](#build-the-update-image)
+  - [Generate the ADU Update Manifest](#generate-the-adu-update-manifest)
+  - [Import the Update Manifest](#import-the-update-manifest)
+  - [Deploy Update](#deploy-update)
+
+## Prerequisite Note
+
+If using WSL to build this project, it is **HIGHLY** recommended to clone the project to the file system inside the WSL instance. Do not clone the repo to your Windows file system and open the WSL working directory to a Windows formatted directory. Clone times, build times, directory max lengths, among other problems may arise.
+
+## Get the middleware
+
+Clone the following repo to download all sample device code, setup scripts, and offline versions of the documentation.
+
+**If you previously cloned this repo in another sample, you don't need to do it again.**
+
+```bash
+git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
```

Comment:
Is it missing the feature branch for now? (-b feature/iot-adu )

Reply:
Yes good call. Will add